src/box/space_def.h

```diff
@@ -130,6 +130,13 @@ struct space_def {
 	/** Number of SQL views which refer to this space. */
 	uint32_t view_ref_count;
 	struct space_opts opts;
+	/**
+	 * Encoding of original (i.e., user-provided) format clause to MsgPack,
+	 * allocated via malloc.
+	 */
+	char *mp_fmt;
```

Comment:
We never use `fmt` for format. Also, `mp_format` sounds like "format of MsgPack", which is confusing. Better rename to `format_data`.

Reply:
Fixed.